Assets/Scripts/Layers/LayerTypes/LayerData.cs

```diff
@@ -76,6 +77,9 @@ public bool ActiveInHierarchy
             }
         }
 
+        [JsonIgnore] public List<LayerPropertyData> LayerProperties => layerProperties;
+        [JsonIgnore] public bool HasProperties => layerProperties.Count > 0;
```

Comment:
Als een alternatief zou je ook `layerProperties.IsEmpty == false` of `!layerProperties.IsEmpty` kunnen gebruiken. Dat doet in principe hetzelfde maar is net wat meer human-readable :)

Reply:
Volgens mij is dat niet C#:
Cannot resolve symbol 'IsEmpty'

Reply:
Je hebt gelijk, ik was in de war met C# zijn ImmutableList: https://learn.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutablelist-1.isempty?view=net-8.0. Never mind :D